lib/errors.js

```diff
@@ -1,39 +1,73 @@
+'use strict';
```

Comment:
I'm pretty sure some code doesn't pass if we were to have "use strict" everywhere.

From a more general point of view, I'm still not convinced that "use strict" is that good. It may help to catch a few things, but they are not hard to catch without, and overall it's takes more time than it save.